gef-ui/server/controllers/unissued-facilities/validation.test.js

```diff
@@ -952,4 +953,78 @@ describe('validation()', () => {
     expect(result.errorsObject.errors).toEqual(expectedErrors);
     expect(result.aboutFacilityErrors).toEqual(expectedFacilityErrors);
   });
+
+  it('should return object with errors populated MIA and coverStartDate over 3 months in future from now', async () => {
+    api.getApplication.mockResolvedValue({ submissionDate: `${getUnixTime(oneYearAgo)}608`, submissionType: CONSTANTS.DEAL_SUBMISSION_TYPE.MIA });
+    mockRequest.body.facilityType = CONSTANTS.FACILITY_TYPE.CASH;
+    mockRequest.body.facilityName = 'UKEF123';
+    mockRequest.body.shouldCoverStartOnSubmission = 'false';
+    mockRequest.query.saveAndReturn = 'true';
+
+    mockRequest.body['issue-date-day'] = format(yesterday, 'd');
+    mockRequest.body['issue-date-month'] = format(yesterday, 'M');
+    mockRequest.body['issue-date-year'] = format(yesterday, 'yyyy');
+
+    mockRequest.body['cover-start-date-day'] = format(fourMonths, 'd');
+    mockRequest.body['cover-start-date-month'] = format(fourMonths, 'M');
+    mockRequest.body['cover-start-date-year'] = format(fourMonths, 'yyyy');
+
+    mockRequest.body['cover-end-date-day'] = format(oneYearFromNow, 'd');
+    mockRequest.body['cover-end-date-month'] = format(oneYearFromNow, 'M');
+    mockRequest.body['cover-end-date-year'] = format(oneYearFromNow, 'yyyy');
+
+    const result = await facilityValidation(mockRequest.body, mockRequest.query, mockRequest.params, {});
+
+    // expected error object format
+    const expectedErrors = {
+      errorSummary: [
+        {
+          href: '#coverStartDate',
+          text: 'The cover start date must be within 3 months of the inclusion notice submission date',
```

Comment:
They do not appear to be correct errors, because application is in `Application` stage so cannot throw `Inclusion` errors?

Reply:
done